hack/build.sh

```diff
@@ -2,16 +2,18 @@
 
 set -e -x -u
 
+VERSION="$(git describe --tags | grep -Eo 'v[0-9]+\.[0-9]+\.[0-9]+' || echo 'develop')"
```

Comment:
any reason to bother to include this in hack/build.sh? it's only used for development.

Reply:
nope. changed.